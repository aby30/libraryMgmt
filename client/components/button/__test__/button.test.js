import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Button from '../index';

afterEach(cleanup);

describe('Button', () => {
    const buttonClick = jest.fn();

    it('renders', () => {
        const { asFragment } = render(<Button type='primary' size='medium' buttonClick={buttonClick} >Button text</Button>);
        expect(asFragment()).toMatchSnapshot();
    })

})