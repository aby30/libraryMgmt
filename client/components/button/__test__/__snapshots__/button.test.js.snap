// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders 1`] = `
<DocumentFragment>
  <button
    class="button primary medium"
  >
    Button text
  </button>
</DocumentFragment>
`;
